Trimble.Modus.Components/Controls/TMInput/TMInput.xaml

```diff
@@ -0,0 +1,119 @@
+
+  <ContentView.Resources>
+    <ResourceDictionary>
+      <converters:ImageNullOrEmptyBoolConverter x:Key="ImageNullOrEmptyBoolConverter" />
+      <converters:StringNotNullOrEmptyBoolConverter x:Key="StringNotNullOrEmptyBoolConverter" />
+      <converters:InverseBoolConverter x:Key="InverseBoolConverter" />
+      <converters:AllTrueConverter x:Key="AllTrueConverter" />
+    </ResourceDictionary>
+  </ContentView.Resources>
+  <Grid x:Name="tmInputContainer"
+        BindingContext="{x:Reference tmInput}"
+        RowDefinitions="auto,auto,auto"
+        RowSpacing="0"
+        Padding="0">
+    <Label x:Name="inputLabel"
+           Grid.Row="0"
+           Padding="0,0,0,5"
+           Text="{Binding TitleText}"
+           VerticalOptions="Center"
+           IsVisible="{Binding TitleText,Converter={StaticResource StringNotNullOrEmptyBoolConverter}}"
+           HorizontalOptions="Start" />
+    <Border x:Name="inputBorder"
+            VerticalOptions="FillAndExpand"
+            Grid.Row="1">
+      <Border.StrokeShape>
+        <Rectangle RadiusX="4"
+                   RadiusY="4" />
+      </Border.StrokeShape>
+      <Grid ColumnDefinitions="auto,*,auto"
+            ColumnSpacing="0"
+            RowSpacing="0" RowDefinitions="40">
+        <Grid.Padding>
+          <OnIdiom Default="5,0"
+                   Phone="16,0" />
+        </Grid.Padding>
+        <ImageButton x:Name="inputLeftIcon"
+                     WidthRequest="20"
+                     HeightRequest="20"
+                     Source="{Binding LeftIconSource}"
+                     Command="{Binding LeftIconCommand}"
+                     CommandParameter="{Binding LeftIconCommandParameter}"
+                     IsVisible="{Binding LeftIconSource,Converter={StaticResource ImageNullOrEmptyBoolConverter}}"
+                     VerticalOptions="CenterAndExpand"
+                     HorizontalOptions="CenterAndExpand"
+                     Grid.Column="0">
+          <ImageButton.IsEnabled>
+            <MultiBinding Converter="{StaticResource AllTrueConverter}">
+              <Binding Path="IsEnabled" />
+              <Binding Path="IsReadOnly"
+                       Converter="{StaticResource InverseBoolConverter}" />
+            </MultiBinding>
+          </ImageButton.IsEnabled>
+        </ImageButton>
+        <controls:BorderlessEntry x:Name="inputBorderlessEntry"
+                                  Grid.Column="1"
+                                  Focused="InputBorderlessEntry_Focused"
+                                  Unfocused="InputBorderlessEntry_Unfocused"
+                                  Placeholder="{Binding Placeholder}"
+                                  Text="{Binding Text}"
+                                  IsPassword="{Binding IsPassword}"
+                                  IsReadOnly="{Binding IsReadOnly}"
+                                  TextColor="{Binding TextColor}"

```

Comment:
Can you remove the TextColor here? We don't have the `TextColor` in the .cs file and it throws a warning now

Reply:
Removed textcolor